src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -456,7 +456,11 @@ static internal bool IsNonAttributedTypeValidForSerialization(Type type)
             { "System.Tuple`8", Array.Empty<string>() },
             { "System.Collections.Queue", new [] {"_syncRoot" } },
             { "System.Collections.Stack", new [] {"_syncRoot" } },
-            { "System.Globalization.CultureInfo", Array.Empty<string>() },
+            { "System.Globalization.CultureInfo", new [] { "m_cultureData", "m_isInherited", "m_nonSortName", "m_parent", "m_sortName" } },
+            { "System.Globalization.CompareInfo", new[] { "m_dataHandle", "m_handleOrigin", "m_sortName" } },
```

Comment:
The fields listed are to be EXCLUDED from serialization, this is the alternative to [NonSerialized] attribute.